de/model.jl

```diff
@@ -0,0 +1,114 @@
+using OrdinaryDiffEq
+using LinearAlgebra
+using DiffEqSensitivity
+using Optim
+using Zygote
+using DelimitedFiles
+
+" Load the experimental data matrix. "
+function get_data(path_RNAseq)
+    # Import RNAseq data as 83 x 84 matrix preprocessed using python
+    exp = DelimitedFiles.readdlm(path_RNAseq, ',', Float64)
+    exp = Matrix(exp)
+    return exp
+end
+
+" Reshape a vector of parameters into the variables we know. "
+@views function reshapeParams(p)
+    w = reshape(p[1:6889], (83, 83))
+    alpha = p[6890:6972]
+    eps = p[6973:7055]
+    
+    @assert length(eps) == 83
+    @assert length(alpha) == 83
+    
+    return w, alpha, eps
+end
+
+" Melt the variables back into a parameter vector. "
+function unshapeParams(w, alpha, eps)
+    return vcat(vec(w), alpha, eps)
+end
+
+" The ODE equations we're using. "
+function ODEeq(du, u, p, t)
+    w, alpha, eps = reshapeParams(p)
+    
+    du .= eps .* (1 .+ tanh.(w * u)) .- alpha .* u
+end
+
+" Solve the ODE system. "
+function solveODE(ps, tps=nothing)
```

Comment:
I *believe* this will now provide you with a time course if you provide the optional argument `tps`. Let me know if you have trouble with it.